packages/net-www/exlibs/vivaldi.exlib

```diff
@@ -73,6 +73,7 @@ vivaldi_src_install() {
     edo mv opt "${IMAGE}/"
     dodir /usr/share
     edo mv usr/share "${IMAGE}/usr/"
+    edo rm -rf "${IMAGE}usr/share/gnome-control-center/"
```

Comment:
Safest way to remove empty dir is to use `rmdir` since it will fail in case if it is not a dir and it will fail if dir is not empty.

It might make sense to remove it from `usr/share` before moving it to target `${IMAGE}`.